docs/source/input_format.rst

```diff
@@ -2,13 +2,71 @@
 Input Format
 ============
 
-The ``opt_einsum`` package is a drop-in replacement for the ``np.einsum`` function
-and supports all input formats that ``np.einsum`` supports.
+The ``opt_einsum`` package is a drop-in replacement for the ``np.einsum``
+function and supports all input formats that ``np.einsum`` supports. There are
+two styles of input accepted, a basic introduction to which can be found in the
+documentation for :func:`numpy.einsum`. In addition to this, ``opt_einsum``
+extends the allowed index labels to unicode or arbitrary hashable, comparable
+objects in order to handle large contractions with many indices.
 
-TBA: "Normal" inputs
 
-In ``opt_einsum``, you can also put anything hashable and comparable such as `str` in the subscript list.
-Note ``np.einsum`` currently does not support this syntax. A simple example of the syntax is:
+'Equation' Input
+----------------
+
+As with :func:`numpy.einsum`, here you specify an equation as a string,
+followed by the array arguments:
+
+.. code:: python
+
+    >>> eq = 'ijk,jkl->li'
+    >>> x, y = np.random.rand(2, 3, 4), np.random.rand(3, 4, 5)
+    >>> z = oe.contract(eq, x, y)
+    >>> z.shape
+    (5, 2)
+
+The main difference is that thousands of symbols in the form of unicode
```

Comment:
Maybe explicitly say "In addition to standard characters, `opt_einsum` also supports unicode..." or the like?

Reply:
Yes good idea.